src/app/server/CommissioningWindowManager.cpp

```diff
@@ -84,8 +84,15 @@ void CommissioningWindowManager::OnPlatformEvent(const DeviceLayer::ChipDeviceEv
     }
     else if (event->Type == DeviceLayer::DeviceEventType::kOperationalNetworkEnabled)
     {
-        app::DnssdServer::Instance().AdvertiseOperational();
-        ChipLogProgress(AppServer, "Operational advertising enabled");
+        CHIP_ERROR err = app::DnssdServer::Instance().AdvertiseOperational();
```

Comment:
I wonder if we ever will have some form of fallback ... since if you cannot advertise operational you probably are not a very useful device. maybe it would be an idea for some device implementations to reboot if this fails or flash some lights as "we are in error state". I think just logging and being void is insufficient.

Or are you suggesting to log and return a CHIP_ERROR? In that case I wonder if we double-handle as it seems not as expected that if you get an error, it was already logged for you.

Reply:
I meant the latter (log and return error), but I don't feel very strongly about it.